QualificationChecker/Controllers/CandidatesController.cs

```diff
@@ -11,100 +13,63 @@ public partial class CandidatesController : ControllerBase
         [HttpPost("showQualified/")]
         public IActionResult ShowQualifiedCandidates([FromBody] MatchRequest request)
         {
-            // Initialize a list to store qualified candidates
-            List<Candidate> qualifiedCandidates = new List<Candidate>();
-
-            //Loop through each candaite and compare answers
-            foreach (var candidate in request.Candidates)
+            var QualifiedCandidates = new List<Candidate>();
```

Comment:
```suggestion
            var qualifiedCandidates = new List<Candidate>();
```
Variables should start with lower case

Reply:
Made lower case for now if I am keeping this variable